schema/text/Table.schema.yaml

```diff
@@ -12,12 +12,5 @@ properties:
     type: array
     items:
       $ref: TableRow.schema.yaml
-  cells:
-    '@id': stencila:cells
-    description: |
-      An array of cells in the table.
-    type: array
-    items:
-      $ref: TableCell.schema.yaml
```

Comment:
Based on your previous comment, and the other referenced schemas, I went ahead and removed the `cells` property from here to better match the XHTML style syntax.

Reply:
:+1: This will require an change in the `xlsx` converter. https://github.com/stencila/convert/blob/master/src/xlsx.ts#L121-L161